primitives/arithmetic/src/biguint.rs

```diff
@@ -439,7 +439,7 @@ impl BigUint {
 }
 
 impl sp_std::fmt::Debug for BigUint {
-	#[cfg(feature = "std")]
+	#[cfg(feature = "serde")]
```

Comment:
Why is the `Debug` implementation required with `serde`?

Reply:
Removed